### PR TITLE
Update and rename mHT-main.py to usage_example.py

### PR DESCRIPTION
**Pull request by nikodem-stolarczyk**
prepared usage example file.

### CODE REVIEW

**Comment by nikodem-stolarczyk** [comment]
Please let me know your suggestions on the example file. We aim to develop a standard example file consistent across all the implementations.

File: python/usage_example.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import mHT 
+from mHT.CPF import cpf_fast, cpf_accurate
+x=1; #dimensionless
+y=1; #dimensionless
+print("the output of the cpf_accurate function")
+print cpf_accurate(x,y)
```
Comment (hjozwiak-umk) [inline python/usage_example.py]: Python 2 syntax, causes an error in Python 3.

 Change to 
`
print(cpf_accurate(x,y))
`

File: python/usage_example.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import mHT 
+from mHT.CPF import cpf_fast, cpf_accurate
+x=1; #dimensionless
+y=1; #dimensionless
+print("the output of the cpf_accurate function")
+print cpf_accurate(x,y)
+
+print("the output of the cpf_fast function")
+print cpf_fast(x,y)
```
Comment (hjozwiak-umk) [inline python/usage_example.py]: Python 2 syntax, causes an error in Python 3.

Change to
`
print(cpf_fast(x,y))
`

**Review by hjozwiak-umk — COMMENTED** [review]
The example looks good! 

We could generate the full profile in one of the cases.

For example
```
nu_range = 3  # or 4
nu_points = 100 # or else
nu_values = np.linspace(-nu_range * GamD, nu_range * GamD, nu_points) 

example_profile = [mHT.profile(nu0,GamD,Gam0_He,Gam2_He,Shift0_He,Shift2_He,NuOptRe_He,NuOptIm_He,nu) for nu in nu_values]
```

**Comment by nikodem-stolarczyk** [comment]
Thank you all for your suggestions and contributions. @hjozwiak-umk please check the latest updates. For me it looks like a final version for the python code. If you confirm this, we can merge it to the main branch.

**Pull request merged**